sdk/typescript/src/builder/bcs.ts

```diff
@@ -74,7 +74,10 @@ export const builder = new BCS(bcs)
     /**
      * Publish a Move module.
      */
-    Publish: { modules: [VECTOR, [VECTOR, BCS.U8]] },
+    Publish: {
+      modules: [VECTOR, [VECTOR, BCS.U8]],
+      dependencies: [VECTOR, BCS.ADDRESS],
```

Comment:
@666lcz, is ADDRESS correct here, or do we have a different marker for `ObjectID`?

Reply:
Just to add some commentary: I believe this is correct for a couple reasons -- I based this off of the `MoveCall` builder above which also expects an object ID (but uses a `BCS.ADDRESS`), there's no `BCS.OBJECT_ID` (I looked lol), but this all makes sense because `ObjectID` is just a newtype around `AccountAddress`.

Reply:
This is correct